src/containers/Section.js

```diff
@@ -84,8 +85,21 @@ const Filter = styled.div`
   }
 `;
 
-const Section = ({ id, mendata, womendata, color, current }) => {
-  console.log(id)
+const Women = styled.div`
```

Comment:
these are logic to define the switching between styling for the buttons depending on what section we are on